platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/AccountsManager.java

```diff
@@ -27,16 +33,48 @@
  * You can access the Mapbox Terms of Service at https://www.mapbox.com/tos/
  */
 class AccountsManager {
+  private static final String TAG = "Mbgl-AccountsManager";
   private static final String PREFERENCE_USER_ID = "com.mapbox.mapboxsdk.accounts.userid";
   private static final String PREFERENCE_TIMESTAMP = "com.mapbox.mapboxsdk.accounts.timestamp";
-  private static final String PREFERENCE_SKU_TOKEN = "com.mapbox.mapboxsdk.accounts.skutoken";
 
   private long timestamp;
+  @Nullable
```

Comment:
Let's ensure that the token is actually not null by fetching it from preferences in the constructor as well.